configure.ac

```diff
@@ -160,6 +96,48 @@ if test x"$HAVE_XRANDR" = "x1"; then
     AC_DEFINE(HAVE_XRANDR)
 fi
 
+# libtasn1
```

Comment:
Can we split these tests up and include them with the commit that actually starts using code from each library?